demos/colab/requirements.txt

```diff
@@ -0,0 +1,10 @@
+torch==1.12.1
+torchvision==0.13.1
+numpy==1.21.6
+pandas==1.3.5
```

Comment:
are pandas and seaborn needed?

Reply:
I removed that, also I can remove the youtube-dl dependency